### DISCUSSION TITLE
How to export data from UI or server?

### DISCUSSION THREAD

**Discussion by DeyangKong** [Arize-ai/phoenix#3514] [Q&A] [answered]
I used this command to start a phoenix server. (**not notebook**)
```
python3 -m phoenix.server.main serve
``` 
Then I opened the web page, and I wanted to export the data, but I didn't see the button.
And I didn't see how to export data from server either.
How could I export the generation data from UI or server?

![image](https://github.com/Arize-ai/phoenix/assets/113083627/ecb18879-f1bf-4cc0-946d-607cc1e68051)

**Comment by mikeldking** [comment, marked as answer]
Hi @DeyangKong! Great to see you here. We have a client and query API that you can use to programmatically download data. https://docs.arize.com/phoenix/tracing/how-to-tracing/extract-data-from-spans
